frontend/src/configs/dev.js

```diff
@@ -0,0 +1,5 @@
+const configs = {
```

Comment:
Do we still need API_URL defined in the env.[ENV] file as well?

Reply:
Ah sorry just saw this comment! No, I think configs.js replaces .eng.[ENV] files, so I deleted .env.development file in this PR 😄

Thank you!!